sysbox-in-docker/Makefile

```diff
@@ -34,7 +34,7 @@ help: ## Show supported docker images
 LAUNCH_INSTRUCTION := docker run -it --privileged --rm --hostname sysbox-in-docker --name sysbox-in-docker \
                 -v /var/tmp/sysbox-var-lib-docker:/var/lib/docker \
                 -v /var/tmp/sysbox-var-lib-sysbox:/var/lib/sysbox \
-                -v /lib/modules/$(uname -r):/lib/modules/$(uname -r):ro \
+                -v /lib/modules/$$(uname -r):/lib/modules/$$(uname -r):ro \
```

Comment:
Why was this necessary (out of curiosity)?

Reply:
I guess you are referring to the double "$". See that we are echo'ying this instruction to stdout, so a "$$" is required to avoid variable interpolation and force command execution instead.